src/test/scala/se/nimsa/dicom/streams/ParseFlowTest.scala

```diff
@@ -447,4 +446,35 @@ class ParseFlowTest extends TestKit(ActorSystem("ParseFlowSpec")) with FlatSpecL
       .expectFragmentsDelimitation()
       .expectDicomComplete()
   }
+
+  it should "parse sequences with VR UN as a block of bytes" in {
+    val unSequence = tagToBytes(Tag.CTExposureSequence) ++ ByteString('U', 'N', 0, 0) ++ intToBytes(24)
+    val bytes = patientNameJohnDoe() ++ unSequence ++ item(16) ++ studyDate()
+
+    val source = Source.single(bytes)
+      .via(new ParseFlow())
+
+    source.runWith(TestSink.probe[DicomPart])
+      .expectHeader(Tag.PatientName)
+      .expectValueChunk()
+      .expectHeader(Tag.CTExposureSequence, VR.UN, 24)
+      .expectValueChunk()
+      .expectDicomComplete()
+  }
+
+  it should "parse sequences with VR UN and implicit VR as a block of bytes" in {
```

Comment:
If VR is implicit how can the non private sequence have VR UN?

Reply:
Right, badly explained. The sequence tag has explicit VR and is UN. The dataset(s) inside the sequence has implicit VR. I'll update the test string.